docs/content/graphql-api-authorization.md

```diff
@@ -0,0 +1,517 @@
+In this case, the integration is hardcoded to ask for `/v1/data/graphqlapi/authz`.
+OPA translates this URL path into a query:
+
+```live:example:query
+data.graphqlapi.authz
+```
+
+The answer returned by OPA for the input above is:
+
+```live:example:output
+```
+
+### 4. Check that `bob` can see `alice`'s salary (because `bob` is `alice`'s manager.)
+
+```shell
+curl --user bob:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"alice\") { salary }}"}'
+```
+
+### 5. Check that `bob` CANNOT see `charlie`'s salary.
+
+`bob` is not `charlie`'s manager, so the following command will fail.
+
+```shell
+curl --user bob:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"charlie\") { salary }}"}'
+```
+
+### 6. Change the policy.
+
+Suppose the organization now includes an HR department. The organization wants
+members of HR to be able to see any salary. Let's extend the policy to handle
+this.
+
+**example-hr.rego**:
+
+```live:hr_example:module:read_only,openable
+package graphqlapi.authz
+
+# Allow HR members to get anyone's salary.
+allowed_query(q) {
+  selected_salary(q)
+  input.user == hr[_]
+}
+
+# David is the only member of HR.
+hr = [
+  "david",
+]
+```
+
+Build a new bundle with the new policy included.
+
+```shell
+opa build example.rego example-hr.rego
+```
+
+The updated bundle will automatically be served by the bundle server, but note that it  might take up to the
+configured `max_delay_seconds` for the new bundle to be downloaded by OPA. If you plan to make frequent policy
+changes you might want to adjust this value in `docker-compose.yml` accordingly.
+
+For the sake of the tutorial we included `manager_of` and `hr` data directly
+inside the policies. In real-world scenarios that information would be imported
+from external data sources.
+
+### 7. Check that the new policy works.
+Check that `david` can see anyone's salary.
+
+```shell
+curl --user david:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"alice\") { salary }}"}'
+curl --user david:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"bob\") { salary }}"}'
+curl --user david:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"charlie\") { salary }}"}'
+curl --user david:password -H "Content-Type: application/json" "localhost:5000/" --data-ascii '{"query":"query { employeeByID(id: \"david\") { salary }}"}'
+```
+
+### 8. (Optional) Use JSON Web Tokens to communicate policy data.
+OPA supports the parsing of JSON Web Tokens via the builtin function `io.jwt.decode`.
+To get a sense of one way the subordinate and HR data might be communicated in the
+real world, let's try a similar exercise utilizing the JWT utilities of OPA.
+
+**example-jwt.rego**:
```

Comment:
Is it possible to compose the policies in some way here? 🤔 Or maybe put a fat marker around "this is new/changed"

Reply:
@srenatus Your suggestions were on the mark. :dart: I was able to refactor the JWT policy down a good bit, and the result is cleaner. The backing module code is hidden now in the way you had suggested. :smile: